### ISSUE TITLE
Make the bots use the q3tourney6 func_bobbing

### ISSUE THREAD

**Issue by coltongit** [zturtleman/spearmint#263]
I don't really know how to code, but shouldn't we have the bots use the func_bobbing in q3tourney6? I think we discussed this in an earlier issue as well.

**Comment by KuehnhammerTobias** [comment]
KuehnhammerTobias didn't show you to solve this issue, he only said that other problems will occur (e.g.: bots will (and sometimes have to) jump even on movers). If the mover is travelling in either x or Y direction, the bot will fall into void.
Redundancy... think we're caught in an endless loop ...:boom:

**Comment by coltongit** [comment]
No, we aren't. Maybe making the bots not jump across the x or ya axis as long as a bot's on the mover(s) will most likely fix this issue.

**Comment by coltongit** [comment]
This seems to be one for the BotMapScripts function @KuehnhammerTobias & @zturtleman.

**Comment by KuehnhammerTobias** [comment]
Really? Well, thats is completely new... wait, I think someone already said this here [#247](https://github.com/zturtleman/spearmint/issues/247)
C'mon do you do this intentionally?.
